code_snippets/python-sdk/inference/diff_baseurl.py

```diff
@@ -22,15 +22,13 @@
 # Example for prediction through URL:
 # model_prediction = model.predict_by_url(URL ,input_type="text")
 
-
 # Example for prediction through Filepath:
 # model_prediction = Model(model_url).predict_by_filepath(image_filepath, input_type="text")
 
-
 model_url = "https://clarifai.com/cohere/embed/models/cohere-embed-english-v3_0"
 
 # You can pass the new base url as paramater while initializing the Model object
-model_prediction = Model(url=model_url, pat="YOUR_PAT",base_url="New Base URL").predict_by_bytes(
+model_prediction = Model(url=model_url, pat="your_pat",base_url="New Base URL").predict_by_bytes(
```

Comment:
What difference does it make? In code_snippets/python-sdk/inference/batch_predict.py it **YOUR_PAT** but here **your_pat**, why?

Reply:
Revert back to YOUR_PAT